views/index.html

```diff
@@ -120,16 +120,16 @@
       <div role="menubar" class="label menu" tabindex="0"><span class="name"><strong><a role="menuitem" class="fake-dropdown button-dropdown" href="#javascriptprocessors">Processor</a></strong></span>
         <div class="dropdown" id="javascriptprocessors">
           <div role="menu" aria-hidden="true" class="dropdownmenu processorSelector" data-type="javascript">
-            <a role="menuitemradio" aria-checked="true" href="#javascript" data-label="JavaScript">JavaScript</a>
-            <a role="menuitemradio" href="#babel">ES6 / Babel</a>
-            <a role="menuitemradio" href="#jsx">JSX (React)</a>
-            <a role="menuitemradio" href="#coffeescript">CoffeeScript</a>
-            <a role="menuitemradio" href="#traceur">Traceur</a>
-            <a role="menuitemradio" href="#typescript">TypeScript</a>
-            <a role="menuitemradio" href="#processing">Processing</a>
-            <a role="menuitemradio" href="#livescript">LiveScript</a>
-            <a role="menuitemradio" href="#clojurescript">ClojureScript</a>
-            <a role="menuitem" href="#convert">Convert to JavaScript</a>
+            <a role="menuitemradio" aria-checked="true" id="#processorSelector-javascript" href="#javascript" data-label="JavaScript">None</a>
+            <a role="menuitemradio" id="#processorSelector-babel" href="#babel">ES6 / Babel</a>
+            <a role="menuitemradio" id="#processorSelector-jsx" href="#jsx">JSX (React)</a>
+            <a role="menuitemradio" id="#processorSelector-coffeescript" href="#coffeescript">CoffeeScript</a>
+            <a role="menuitemradio" id="#processorSelector-traceur" href="#traceur">Traceur</a>
```

Comment:
Can you rename these `id`s to `#processor-traceur` (losing the "selector" part)?